gamestonk_terminal/cryptocurrency/crypto_controller.py

```diff
@@ -7,36 +8,62 @@
 from gamestonk_terminal import feature_flags as gtff
 from gamestonk_terminal.helper_funcs import get_flair
 from gamestonk_terminal.menu import session
-from gamestonk_terminal.cryptocurrency import pycoingecko_view
-from gamestonk_terminal.cryptocurrency import coinmarketcap_view as cmc_view
+from gamestonk_terminal.cryptocurrency import (
+    binance_model,
+    pycoingecko_view,
+    coinmarketcap_view as cmc_view,
+)
 
 
 class CryptoController:
 
-    CHOICES = ["help", "q", "quit", "load", "view", "top"]
+    CHOICES = [
+        "help",
+        "q",
+        "quit",
+        "load",
+        "view",
+        "top",
+        "trend",
+        "book",
+        "trades",
+        "candle",
+        "balance",
+        "add",
+    ]
 
     def __init__(self):
         """CONSTRUCTOR"""
 
         self.crypto_parser = argparse.ArgumentParser(add_help=False, prog="crypto")
         self.crypto_parser.add_argument("cmd", choices=self.CHOICES)
         self.current_coin = None
+        self.current_currency = None
         self.current_df = pd.DataFrame()
 
     @staticmethod
-    def print_help(current_coin):
+    def print_help(cont):
```

Comment:
Here you shouldn't expect any outside argument. Instead, you want to use arguments from the class you are in. So this should be "self"

Reply:
So `self` raises a pylint error (not sure why here and not in others) (Something about self in static method).  Since that doesn't assign anything to the object, that is just accepting the object (which is a controller object).

Reply:
I think you need to add @staticmethod above or so, there are other controllers that use members of the class, you can check those. E.g. research controller.

Regarding your comment that's true, it's just a parameter, but "_self is a parameter in function and the user can use a different parameter name in place of it. Although it is advisable to use self because it increases the readability of code._" https://www.edureka.co/blog/self-in-python/

Reply:
It is wrapped as a static method.  And I pretty much copied it over so I am unsure why self raises an error in this case.

Reply:
Let me see if I can figure it out. 

Do you remember the warning you were getting?

Reply:
The error is `W0211 Static method with 'self' as first argument (bad-staticmethod-argument)`

Reply:
Can it just be a non static method?

Reply:
Got it.

You're trying to do this:
```
crypto_controller.print_help(crypto_controller)
```
when you should do this:
```
crypto_controller.call_help(None)
```